server/chat-plugins/customavatars.js

```diff
@@ -0,0 +1,106 @@
+//Original plugin from https://github.com/CreaturePhil/Showdown-Boilerplate/blob/master/chat-plugins/customavatar.js.
+//Credits to CreaturePhil and the other listed contributors.
+//updated for the main server by Maxalexanderpi, with help from Hoeenhero.
+'use strict';
+
+/** @type {typeof import('../../lib/fs').FS} */
+const FS = require(/** @type {any} */('../../.lib-dist/fs')).FS;
+
+const path = require('path');
+const fs = require('fs');
+
+/* eslint no-restricted-modules: [0] */
+const AVATAR_PATH = 'config/avatars/';
+
+const VALID_EXTENSIONS = ['.png'];
```

Comment:
Since it contains one value, is there any reason you're using an array instead of a string?